src/ASTBuilder.ts

```diff
@@ -1583,26 +1585,24 @@ export class ASTBuilder
       unitAlias,
       unitAliasIdentifier,
       symbolAliases,
-      symbolAliasesIdentifiers,
+      symbolAliasesIdentifiers
     }
 
     return this._addMeta(node, ctx)
   }
 
   public buildEventParameterList(ctx: SP.EventParameterListContext) {
-    return ctx.eventParameter().map((paramCtx: any) => {
+    return ctx.eventParameter().map((paramCtx: SP.EventParameterContext) => {
```

Comment:
`ctx.eventParameter()` returns a `SP.EventParameterContext[]` so we can assume that the map will iterate over `SP.EventParameterContext`

Reply:
This and the previous snippet actually are inferred correctly, so we can just remove `: any` instead of explicitly using this type. I pushed a commit doing that.